src/main/java/pl/betoncraft/betonquest/database/PlayerData.java

```diff
@@ -405,11 +405,19 @@ public void setBackpack(List<ItemStack> list) {
 		}
 	}
 
+	/**
+	 * Get the entries.
+	 * @return entries of current 
+	 */
+	public List<Pointer> getEntries() {
+		return entries;
```

Comment:
I'dd suggest returning an Unmodifiable list instead because if the list returned by this method is edited this will screw up some internal mechanics.
So my advise here would be:
```java
public List<Pointer> getEntries() {
	return Collections.unmodifiableList(entries);
}
```